src/main/java/au/org/emii/geoserver/extensions/filters/layer/data/Filter.java

```diff
@@ -19,7 +19,7 @@
     private Boolean enabled;
     private Boolean visualised;
     private Boolean excludedFromDownload;
-    public HashMap extras = new HashMap();
+    public Map<String, String> extras = new HashMap<String, String>();
```

Comment:
For consistency I'd suggest we instantiate this variable in the constructor with the other fields.